src/nasa_page.tsx

```diff
@@ -1,19 +1,15 @@
 import React from "react";
+import Info from "./info_page_template";
+import logo from './logo.svg';
 
-type NasaProps = {
-    title: string;
-}
 
-const Nasa: React.FC<NasaProps> = ({
-    title,
-}) => {
-    return (
-        <body>
-        <h1>{title}</h1>
-        <p>This is a paragraph.</p>
-        <p>This is a second paragraph.</p>
-        </body>
-    );
-}
+const Nasa = <Info
```

Comment:
Nice refactoring - though this is no longer a component. We'd still want to keep Nasa as a component, so something like 
```suggestion
const Nasa: React.FC<> = () => {
     return (
        <div className="Nasa">
            <Info .... />
        </div>
    )
}
```